### PR TITLE
Draw precincts

### PR DESCRIPTION
**Pull request by nabohall**
All the precinct stuff is done. Coloring, popup, and pco info per precinct

### CODE REVIEW

File: js/map.js [hunk 1]
```diff
@@ -41,4 +51,48 @@ var GeoCodeAddress = function() {
       alert('Geocode was not successful for the following reason: ' + status);
     }
   });
-}
\ No newline at end of file
+}
+
+var DrawAllPrecincts = function() {
+  var url = "https://xv1zwa82p7.execute-api.us-east-1.amazonaws.com/dev/precincts";
```
Comment (corespace) [inline js/map.js]: Let's use https://api.34dems.net/precincts so we keep my bill for lambda low :)

File: js/map.js [hunk 2]
```diff
@@ -41,4 +51,48 @@ var GeoCodeAddress = function() {
...
+    var pcoContent = '<div id="content">'
+      + '<h3>Precinct: ' + feature.getProperty('name') + '</h3>';
+    var pcoInfo = feature.getProperty('pco');
+
+    if(pcoInfo) {
+      pcoContent = pcoContent  +  
+      '<h5>' + pcoInfo['first'] + ' ' + pcoInfo['last'] + '</h5>';
+    } else {
+      pcoContent = pcoContent + '<p> No PCO currently.</p>' + 
+        '<a href="http://www.34dems.org/get-active/pco/"> Click here to be appointed!</a>';
```
Comment (corespace) [inline js/map.js:92]: Would we want to open this in a new window?

**Review by corespace — COMMENTED** [review]
Looking good!

File: index.html [hunk 3]
```diff
@@ -14,7 +14,7 @@
   </head>
   <body onload="InitMap()">
     <div id="map"></div>
-    <div>
+    <div id="address">
       <input type="textbox" name="Address" id="address">
```
Comment (corespace) [inline index.html]: Here's the problem - both the div and the input have id 'address'

File: data/1481.json [hunk 4]
```diff
@@ -0,0 +1,117 @@
+{
```
Comment (corespace) [inline data/1481.json:1]: Let's not check this file in.

File: index.html (hunk 3 above)
Comment (corespace) [reply index.html]: I see why you wanted this - so the style puts it in the same place as the map. Just use another string and it'll be fine :)

File: js/map.js [hunk 5]
```diff
@@ -41,4 +51,48 @@ var GeoCodeAddress = function() {
...
+}
+
+var DrawAllPrecincts = function() {
+  var url = "https://xv1zwa82p7.execute-api.us-east-1.amazonaws.com/dev/precincts";
+
+  globals.GetMap().data.loadGeoJson(url);
+  //Set any precinct that has no PCO to red
+  globals.GetMap().data.setStyle(function(feature) {
+    var pco = feature.getProperty('pco');
+    var color = pco ? 'white' : 'red';
```
Comment (corespace) [inline js/map.js]: Carrie's requesting we use '#dd9933' instead of 'red'

File: js/map.js (hunk 2 above)
Comment (nabohall) [reply js/map.js:92]: We're trying to not do that unless it redirects away from the site. It makes it less accessible for visually impaired visitors.

**Review by corespace — APPROVED** [review]

**Pull request merged**